src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -978,10 +973,9 @@ public Date extractEventDateFrom(final IBaseDataObject d, final boolean lastReso
             if (value != null) {
                 try {
                     date = emissary.util.TimeUtil.getDateFromISO8601(value);
-                    logger.debug("Successfully parsed event date from " + paramName);
                     return date;
                 } catch (DateTimeParseException ex) {
-                    logger.debug("Cannot parse EventDate " + value);
+                    logger.debug("Cannot parse EventDate", ex);
```

Comment:
Might need to get back to you on this one.

Reply:
For this one I assume just using the exception message in the log would work rather than calling the exact "value"